src/util/type.cpp

```diff
@@ -12,7 +12,6 @@ Author: Daniel Kroening, kroening@kroening.com
 
 #include "type.h"
 #include "std_types.h"
-#include "namespace.h"
```

Comment:
I think (and this would make me rather happy) `std_types.h` could go as well? That finally resolves a dependency that should not have existed in the first place...